sway-lib-std/src/vec.sw

```diff
@@ -447,3 +449,10 @@ impl<T> From<raw_slice> for Vec<T> {
         asm(ptr: (self.buf.ptr(), self.len)) { ptr: raw_slice }
     }
 }
+
+#[test()]
+fn test_vec_with_len_1() {
```

Comment:
Can we have some assertions (pre-post conditions) to turn this into a test? Atm this isn't technically a test

Reply:
Added some here https://github.com/FuelLabs/sway/pull/4445/commits/8df9b1fdfeb20ae26dc411a3218c5aa10f8234ac